src/Language/Haskell/Liquid/GHC/Interface.hs

```diff
@@ -339,7 +339,7 @@ updLiftedSpec :: Ms.BareSpec -> Maybe Ms.BareSpec -> Ms.BareSpec
 updLiftedSpec s1 Nothing   = s1 
 updLiftedSpec s1 (Just s2) = (clear s1) `mappend` s2 
   where 
-    clear s                = s { sigs = [], aliases = [], ealiases = [], qualifiers = [] }
+    clear s                = s { sigs = [], aliases = [], ealiases = [], qualifiers = [], dataDecls = [] }
```

Comment:
Why is it ok to wipe the `dataDecls`? Is it because the data-constructor signatures are still kept so we don't really need the datatype?

Reply:
No, it is because the dataDecls of the parsed file are not qualified. 
In my example, e.g., they use cProp instead of C.cProp. 
So I have to remove the parsed ones and replace them with the qualified ones....